modules/covid_vaccine/app/serializers/covid_vaccine/v0/expanded_registration_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module CovidVaccine
+  module V0
+    class ExpandedRegistrationSerializer < ActiveModel::Serializer
+      attribute :created_at
+
+      def id
+        nil
```

Comment:
I'm curious why id should be nil and not the record id?

Reply:
i don't want to leak any record identifiers back to the unauthenticated client - especially not a sequential DB ID. The submission_uuid could be used here I guess, but also - what ever is going to be done with that value? nothing. Maybe some miniscule value for debugging.